.github/actions/setup-woocommerce-monorepo/action.yml

```diff
@@ -21,6 +21,35 @@ runs:
           uses: 'pnpm/action-setup@a3252b78c470c02df07e9d59298aecedc3ccdd6d'
           with:
               version: ${{ steps.read-pnpm-version.outputs.version }}
+        - name: 'Get PNPM store directory'
+          shell: 'bash'
+          run: |
+              echo "PNPM_STORE_PATH=$(pnpm store path --silent)" >> $GITHUB_ENV
+        - name: 'Cache PNPM Dependencies'
+          if: ${{ inputs.build == 'false' }}
+          uses: 'actions/cache@ab5e6d0c87105b4c9c2047343972218f562e4319'
+          with:
+            path: ${{ env.PNPM_STORE_PATH }}
+            key: "${{ runner.os }}-pnpm-${{ hashFiles( '**/pnpm-lock.yaml' ) }}"
+            restore-keys: '${{ runner.os }}-pnpm-'
+        - name: 'Cache Playwright Downloads'
+          if: ${{ inputs.build != 'false' }}
+          uses: 'actions/cache@ab5e6d0c87105b4c9c2047343972218f562e4319'
+          with:
+            path: '~/.cache/ms-playwright/'
+            key: "${{ runner.os }}-playwright-${{ hashFiles( '**/pnpm-lock.yaml' ) }}"
+            restore-keys: '${{ runner.os }}-playwright-'
+        - name: 'Cache build folders (mimics incremental builds)'
```

Comment:
It is not needed; build times are still varying as they were before. There are no visible results for keeping this cache.